Shell/TheoryAxioms.cpp

```diff
@@ -276,7 +275,7 @@ void TheoryAxioms::addOrderingTotality(Interpretation less)
   TermList srt = theory->getOperationSort(less);
   Literal* eq = Literal::createEquality(true,x,y,srt);
 
-  addTheoryClauseFromLits({l12,l21,eq}, InferenceRule::THA_ORDER_TOTALALITY, CHEAP);
+  addTheoryClauseFromLits({l12,l21,eq}, InferenceRule::THA_ORDER_TOTALITY, CHEAP);
```

Comment:
Thank you thank you thank you. I've been getting brain damage for years from these identifiers.